.github/workflows/admin.yml

```diff
@@ -33,20 +33,14 @@ jobs:
 
     env:
       GO_VERSION: ${{ matrix.go-version }}
-      GOPATH: ${{ github.workspace }}
-      GOBIN: ${{ github.workspace }}/bin
-      WORKING_DIR: ${{ github.workspace }}/src/github.com/percona/pmm
 
     defaults:
       run:
-        working-directory: ${{ env.WORKING_DIR }}/admin
+        working-directory: ${{ github.workspace }}/admin
 
     steps:
-      - name: Check out code into the working directory under the GOPATH
+      - name: Check out code
         uses: percona-platform/checkout@v3
-        with:
```

Comment:
what is the reason to remove it? I think it speeds up CI work.

Reply:
Because it's default value: https://github.com/actions/checkout#usage